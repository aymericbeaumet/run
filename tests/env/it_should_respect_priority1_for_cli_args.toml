env = [ "__RUN_CLI_FOOBAR__=from_main_block" ]

[[run]]
env = [ "__RUN_CLI_FOOBAR__=from_run_block" ]
name = "p1"
cmd = ["bash", "-c", "echo $__RUN_CLI_FOOBAR__"]