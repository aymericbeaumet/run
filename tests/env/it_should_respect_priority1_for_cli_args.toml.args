-e
__RUN_CLI_FOOBAR__=from_cli_args