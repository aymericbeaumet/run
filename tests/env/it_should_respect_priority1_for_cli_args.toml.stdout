[p1] from_cli_args